app/models/school.rb

```diff
@@ -0,0 +1,13 @@
+class School < ApplicationRecord
+  belongs_to :local_authority, required: true
+
+  enum phase: {not_applicable: 0, nursery: 1, primary: 2,
+               middle_deemed_primary: 3, secondary: 4,
+               middle_deemed_secondary: 5, sixteen_plus: 6, all_through: 7,}
+
+  validates :urn, presence: true
+  validates :name, presence: true
+  validates :school_type_group_code, presence: true
```

Comment:
Do we want to have `enum`s defined on these types also to provide some level of validation for the values?